indico/modules/events/sessions/templates/management/session_person_list.html

```diff
@@ -0,0 +1,93 @@
+                <label for="filter-primary" class="i-button">{% trans %}Primary authors{% endtrans %}</label>
+                <input type="checkbox" name="authors" id="filter-secondary" data-filter="secondary" checked>
+                <label for="filter-secondary" class="i-button">{% trans %}Co-authors{% endtrans %}</label>
+            </div>
+        </div>
+    </div>
+    <div>
+        <table class="i-table-widget tablesorter">
+            <thead>
+                <tr class="i-table">
+                    <th class="i-table checkbox-column" data-sorter="false"></th>
+                    <th class="i-table">{% trans %}Name{% endtrans %}</th>
+                    <th class="i-table">{% trans %}Email{% endtrans %}</th>
+                    <th class="i-table">{% trans %}Affiliation{% endtrans %}</th>
+                    <th class="i-table">{% trans %}Author type{% endtrans %}</th>
+                    <th class="i-table" data-sorter="false"></th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for session_person in session_persons -%}
+                    {% set event_person = session_person.person %}
+                    {% set is_speaker = session_person.is_speaker %}
+                    {% set is_primary_author = session_person.author_type.name == 'primary' %}
+                    {% set is_secondary_author = session_person.author_type.name == 'secondary' %}
+                    {% set person_data = dict(speaker=is_speaker,
+                                              primary=is_primary_author,
+                                              secondary=is_secondary_author) %}
+                    <tr class="i-table" data-person-roles="{{ person_data | tojson | forceescape }}">
+                        <td class="i-table">
+                            <input type="checkbox" value="{{ event_person.id }}" class="select-row"
+                                   name="person_id">
+                        </td>
+                        <td class="i-table">{{ event_person.full_name }}</td>
+                        <td class="i-table email-column">{{ event_person.email }}</td>
+                        <td class="i-table">{{ event_person.affiliation }}</td>
+                        <td class="i-table">
+                            {%- if is_primary_author %}
+                                {%- trans %}Primary{% endtrans -%}
+                            {% elif is_secondary_author %}
+                                {%- trans %}Secondary{% endtrans -%}
+                            {% else %}
+                                {%- trans %}None{% endtrans -%}
+                            {% endif -%}
+                        </td>
+                        <td class="i-table">
+                            <i class="icon-mic speaker-icon" style="color: {{ '#0B8D00' if is_speaker else '#BBB' }}"
+                               title="{% trans %}Whether this person is a speaker {% endtrans %}"></i>
```

Comment:
I'd rather add extra icons for the speaker type and then gray out the ones that do not apply. then the title is not needed anymore (if you keep it, use different titles depending on `is_speaker`)